meos/include/meos.h

```diff
@@ -1948,7 +1952,7 @@ extern bool tpoint_is_simple(const Temporal *temp);
 extern double tpoint_length(const Temporal *temp);
 extern Temporal *tpoint_speed(const Temporal *temp);
 extern int tpoint_srid(const Temporal *temp);
-extern STBox *tpoint_stboxes(const Temporal *temp, int *count);
+extern STBox *tpoint_stboxes_from_segs(const Temporal *temp, int max_count, int *count);
```

Comment:
I thought that the suffix `_from_segs` would be used together with the `segs_per_box` parameter. I don't think it fits well with a function where you specify the max number of boxes. The old name `tpoint_stboxes` seems to be better in this case.